test/testcafe/spec/EnrollAuthenticatorPhoneView_spec.js

```diff
@@ -167,9 +167,9 @@ test
     const enrollPhonePageObject = await setup(t);
     await checkA11y(t);
     await enrollPhonePageObject.clickNextButton();
-    await t.expect(await enrollPhonePageObject.resendCodeExists(2)).eql(false);
+    await t.expect(await enrollPhonePageObject.resendCodeExists(1)).eql(false);
```

Comment:
Just curious, why the nth index was 2 and now it should equal 1?

Reply:
These tests started failing maybe due to some flakiness with timers and the resend code alert not appearing right away. I've also tested locally and seems like the form-level "We found some errors" pop up does not always appear when testcafe tries to simulate clicking the "Verify" button. Regardless, I think the resend code should always at least be the 0th or 1st index anyway if you see the screenshot. I think since the original test was only asserting the presence of an element with `role="alert"` at the 2nd index and not the content of that element, that was actually asserting the presence of the field-level "This field cannot be left blank"
<img width="433" alt="Screenshot 2024-02-02 at 5 45 37 PM" src="https://github.com/okta/okta-signin-widget/assets/106110543/ec52d22e-a694-4b2b-8e62-b5cb0378a38c">